net-analyzer/greenbone-security-assistant/greenbone-security-assistant-7.0.3.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR="emake"
+inherit cmake-utils systemd
+MY_PN="gsa"
+
+DESCRIPTION="Greenbone Security Assistant for OpenVAS"
+HOMEPAGE="http://www.openvas.org/"
+SRC_URI="https://github.com/greenbone/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-2+ BSD MIT"
+KEYWORDS="~amd64 ~x86"
+IUSE="+extras"
+
+DEPEND="
+	dev-libs/libgcrypt:0=
+	dev-libs/libxml2:2
+	dev-libs/libxslt
+	extras? (
+		dev-python/polib
+		sys-devel/gettext
+	)
+	>=net-analyzer/openvas-libraries-9.0.3
+	net-libs/gnutls:=[tools]
+	net-libs/libmicrohttpd[messages]"
+
+RDEPEND="
+	${DEPEND}"
+
+BDEPEND="
+	app-doc/doxygen[dot]
```

Comment:
You probably want to make this conditional with extras? USE flag, as you've done with all other ebuilds.

Reply:
You are right.Fixed.